src/helpers/proofs/attestor.ts

```diff
@@ -47,8 +47,8 @@ export async function getEddsaPublicKey() {
   return data
 }
 
-export function sendEmail(email: string) {
+export function sendEmails(emails: readonly string[]) {
```

Comment:
`readonly` is because we read emails using `useSnapshot`, which returns `readonly` values

Reply:
Anyway, we don't mutate the `emails` here, so the `readonly` is a good modifier